runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -255,6 +304,9 @@ static void fork_nb_large_wrapper(fork_t* f) {
   void* f_arg;
   chpl_memcpy(&f_arg, f->arg, sizeof(void*));
 
+  // MPF - should internal to the GASNet comms layer communication
+  // like this be counted in e.g. chpl_comm_commDiagnostics.get ?
+  // If not, we probably want to make e.g. do_remote_get.
```

Comment:
Also, isn't this a source of test system noise, because different comms layers emit different numbers of GETs?